scripts/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This may require and ADR (I think it should), but could we maybe try and stick to this convention: https://github.blog/2015-06-30-scripts-to-rule-them-all  ?

Reply:
@anneschuth. To be honest i have never seen this setup of script conventions in a repo. Do you have some examples of big repos following this? To me this setup does not feel very intuitive.

Reply:
The most prominent example ofcourse is GitHub itself, they use this internally.
The point is more to have a convention rather than that it needs to be this one. That said, I've used this before (albeit a bit loosely) and it helped in knowing what to expect/do in a less familiar repo.

Reply:
What do you think about this?

- rename `scripts/test.sh` to `script/test`  (and add `chmod +x`) 
- move typecheck into this 
- start with linting

Reply:
I really like this setup that GitHub uses. So for what it is worth, I upvote that! :)